cmd/analyzer/analyzer.go

```diff
@@ -512,6 +526,16 @@ func (a *Service) Start() {
 	ctx, cancelAnalyzers := context.WithCancel(context.Background())
 	defer cancelAnalyzers() // Start() only returns when analyzers are done, so this should be a no-op, but it makes the compiler happier.
 
+	// Start helpers.
+	for _, helper := range a.helpers {
```

Comment:
Should probably also try to gracefully `Shutdown()` the server helpers in `received interrupt, shutting down` part. And call `Close()` on the servers in `cleanup`.

Reply:
Today's end-to-end tests are tomorrow's production features :grin: